sdks/wasm/DebuggerTestSuite/Tests.cs

```diff
@@ -123,6 +123,70 @@ public class SourceList : DebuggerTestBase {
 			});
 		}
 
+		[Fact]
+		public async Task CheckBreakpoint () {
+			var insp = new Inspector ();
+
+			var scripts = SubscribeToScripts(insp);
+
+			await Ready ();
+			await insp.Ready (async (cli, token) => {
+				ctx = new DebugTestContext (cli, insp, token, scripts);
+
+				var bp1_res = await SetBreakpoint ("dotnet://debugger-test.dll/debugger-array-test.cs", 197, 44);
+				Assert.EndsWith ("debugger-array-test.cs", bp1_res.Value["breakpointId"].ToString());
+				Assert.Equal (1, bp1_res.Value ["locations"]?.Value<JArray> ()?.Count);
+
+				var loc = bp1_res.Value ["locations"]?.Value<JArray> ()[0];
+
+				Assert.NotNull (loc ["scriptId"]);
+				Assert.Equal (197, loc ["lineNumber"]);
+				Assert.Equal (44, loc ["columnNumber"]);
+
+				var bp2_res = await SetBreakpoint ("dotnet://debugger-test.dll/debugger-array-test.cs", 197, 49);
+				Assert.EndsWith ("debugger-array-test.cs", bp2_res.Value["breakpointId"].ToString());
+				Assert.Equal (1, bp2_res.Value ["locations"]?.Value<JArray> ()?.Count);
+
+				var loc2 = bp2_res.Value ["locations"]?.Value<JArray> ()[0];
+
+				Assert.NotNull (loc2 ["scriptId"]);
+				Assert.Equal (197, loc2 ["lineNumber"]);
+				Assert.Equal (49, loc2 ["columnNumber"]);
+
+			});
+		}
+
+		[Fact]
+		public async Task Create0Breakpoint () {
+			var insp = new Inspector ();
+
+			var scripts = SubscribeToScripts(insp);
+
+			await Ready ();
+			await insp.Ready (async (cli, token) => {
+				ctx = new DebugTestContext (cli, insp, token, scripts);
+
+				var bp1_res = await SetBreakpoint ("dotnet://debugger-test.dll/debugger-array-test.cs", 197, 0);
```

Comment:
The column number here is the only difference between the tests, correct? You can use `[InlineData (..)]` for that. See other uses of `InlineData` in the tests.

Reply:
Also, you might want to add another test that has multiple possible breakpoints on the same line, so when you use `col==0`, you get multiple breakpoints set.

Reply:
Ignore that last comment!